containers/proxy-tftpd-image/tftp_wrapper.py

```diff
@@ -212,11 +212,14 @@ def get_response_data(self):
         # accept only mac based config and defaults file
         if path.startswith("pxelinux.cfg/01-"):
             # request specific system configuration
-            logging.debug(f"Got request for system PXE {path}, forwarding as is")
-            return HttpResponseData(f"{target}/tftp/{path}", capath)
+            logging.debug(f"Got request for system PXE {path}, filtering HTTP")
+            return HttpResponseDataFilteredPXE(
+                f"{target}/tftp/{path}", capath, self._proxyFqdn, self._serverFqdn
+            )
         elif path.startswith("grub/system"):
-            logging.debug(f"Got request for system GRUB {path}, forwarding as is")
-            return HttpResponseData(f"{target}/tftp/{path}", capath)
+            logging.debug(f"Got request for system GRUB {path}, filtering HTTP")
+            return HttpResponseDataFilteredGrub(
+                f"{target}/tftp/{path}", capath, self._proxyFqdn, self._serverFqdn
```

Comment:
Missing close parenthesis.

Reply:
added the missing parenthesis

Reply:
Don't worry, @cbosdo already fixed that.